README.md

```diff
@@ -24,6 +24,23 @@ an Apache [Thrift] service so that users can use their preferred programming
 language to communicate with Accumulo (provided that language has a supported
 Thrift language binding).
 
+# Running the Accumulo proxy
+
+1. Build the proxy tarball and install it.
+
+    ```
+    cd /path/to/accumulo-proxy
+    mvn clean package
+    tar xzvf ./target/accumulo-proxy-2.0.0-SNAPSHOT-bin.tar.gz -C /path/to/install
+    ```
+
+2. Edit `proxy.properties` and `accumulo-client.properties` and run the proxy.
+
+    ```
+    cd /path/to/install/accumulo-proxy-2.0.0-SNAPSHOT
+    ./bin/accumulo-proxy -p conf/proxy.properties -c $ACCUMULO_HOME/conf/accumulo-client.properties
+    ```
+
```

Comment:
It would be nice to put all the props to run the proxy into a single properties file. It makes specifying the config much easier, and the file itself can be easily organized (with line breaks) to separate the properties that are specific to the proxy from those that are general client properties.

Reply:
That's a good idea. I'll do that

Reply:
This was done